lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -254,6 +254,26 @@
         .course-container{
           border: 1px solid $border-color-l4;
           border-radius: 3px;
+          .xseries-container{
+            margin: ($baseline*0.4) ($baseline/5) 0 ($baseline*0.4);
+
+            .xseries-icon{
+                @include float(left);
```

Comment:
Shouldn't need float and the display:inline here, though I can take a look on a sandbox once that exists if that's easier.